src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -418,6 +427,25 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                                custom_locations_oid, helm_client_location, enable_private_link, arm_metadata, onboarding_timeout, container_log_path)
     return put_cc_response
 
+def validate_existing_provisioned_cluster_for_reput(cluster_resource, kubernetes_distro, kubernetes_infra, enable_private_link, private_link_scope_resource_id, distribution_version, azure_hybrid_benefit, location):
+    if (cluster_resource != None) and (cluster_resource.kind.lower() == "provisionedcluster"):
```

Comment:
It is recommended to set `provisionedcluster` to a constant, which will facilitate unified modification in the future.

Reply:
updated the PR and made those changes